### PR TITLE
`knausj`: rebrand, honor original creator

### PR DESCRIPTION
**Pull request by phillco**
https://github.com/talonhub/community/issues/935. We're not in ~Kansas~ knausj anymore...

This updates:

- The main title of the README (as well as most references therein) and adds a note about our origins. [View here]( https://github.com/talonhub/community/tree/rebrand)
    - The cloning instructions in the README.
        - (This might be up for debate; do we want people to `git clone https://github.com/talonhub/community community`? Some users might end up with two copies. I think @lunixbochs had a plan to check for if people have both knausj and community cloned duplicative side by side by mistake.)
        - GitHub's web UI recommends `https://github.com/talonhub/community.git` as the HTTPS URL, but our old instructions did not include the `.git` suffix. What do we want to do?
- All GitHub links in the form of `https://github.com/knausj85/knausj_talon/`, such as to issues.
- A few manual references in text files, such as in the deprecation process.

### CODE REVIEW

**Comment by phillco** [comment]
I'll update the other references, one sec

File: README.md [hunk 1]
```diff
@@ -76,17 +81,17 @@ If you use vim, just start with the numbers and alphabet, otherwise look at gene
 ### Alphabet
```
Comment (pokey) [inline README.md:81]: Why did you include the `.git` here? I would have just done a find-replace to community without the `.git`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
-# knausj_talon
+# community
```
Comment (pokey) [inline README.md:1]:
```suggestion
# Talon community grammar
```

**Review by pokey — COMMENTED** [review]
Looks good with minor comments

File: README.md (hunk 1 above)
Comment (phillco) [reply README.md:86]: You are quite right. Fixed.

Comment (phillco) [reply README.md:86]: For HTTPS cloning, GitHub actually recommends `https://github.com/talonhub/community.git` for the HTTPS URL, but it works without the `.git`  and the old way did not include the `.git`, so I removed it, but not sure what the best way is.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+
+> **Note**
+>
+> Originally called `knausj_talon`, after [its original creator :superhero:](https://github.com/knausj85).
+>
+> You might see the reference to the original name in places. If you had the old name cloned, you don't have to do anything; GitHub will redirect the old name.
+>
+> However, if you'd like, you can update the remote URL in your existing local repo:
+>
+> `cd ~/.talon/knausj_talon && git remote set-url origin https://github.com/talonhub/community`
```
Comment (pokey) [inline README.md:13]: I don't think we want so much about the original at the top here. I'd put the update steps in a separate section / file and just link to it here so it doesn't distract / confuse new users

Comment (knausj85) [reply README.md:13]: Agree

**Review by knausj85 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (phillco) [reply README.md:1]: I'm kind of used to the GitHub convention of the first line being the name of the repository, with a description both underneath and in the repository description field. I don't feel strongly but I thought this was the convention unless you had a strong reason not to

File: README.md (hunk 3 above)
Comment (phillco) [reply README.md]: Done

File: apps/vscode/command_client/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ This directory contains the client code for communicating with the [VSCode comma
 
 ## Contributing
 
-The source of truth is in https://github.com/knausj85/knausj_talon/tree/main/apps/vscode/command_client, but the code is also maintained as a subtree at https://github.com/pokey/talon-vscode-command-client.
+The source of truth is in https://github.com/talonhub/community/tree/main/apps/vscode/command_client, but the code is also maintained as a subtree at https://github.com/pokey/talon-vscode-command-client.
 
 To contribute, first open a PR on knausj. Once the PR is merged, you can push the changes to the subtree by running the following commands on an up-to-date knausj main: (need write access)
```
Comment (auscompgeek) [inline apps/vscode/command_client/README.md:9]: This needs updating too.

Comment (phillco) [reply apps/vscode/command_client/README.md]: Done, thanks

File: README.md (hunk 2 above)
Comment (knausj85) [reply README.md:1]: This seems to be the general convention still, I think it's fine as is.

**Comment by knausj85** [comment]
This looks good to me, thanks.

**Pull request merged**